sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -5,6 +5,8 @@ package com.azure.cosmos.spark
 
 // cosmos db related constants
 private object CosmosConstants {
+  val currentVersion = "3.0.1_4.0-3.0.0" // TODO: Define a version format
```

Comment:
we already have the version in the pom file. 

Could you try to use this to rely on the version in the pom file:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java#L269

Reply:
Done